src/app/components/filter/index.html

```diff
@@ -96,8 +96,7 @@ <h3>Inline Filters</h3>
       <div class="sky-subsection-heading">Inline filters</div>
       <p>
         Use inline filters to apply up to three simple filters to a <a routerLink="/components/list">list</a> or
-        <a routerLink="/components/tiles">tile</a>. This pattern works well with <a routerLink="/components/checkbox">checkboxes</a>
-        or <a routerLink="/components/dropdown">dropdowns</a>, but not with complex selection controls.
+        <a routerLink="/components/tiles">tile</a>. This pattern works well with <a routerLink="/components/checkbox">checkboxes</a>, HTML <sky-code>select</sky-code> fields, or SKY UX <a routerLink="/components/select-field">select fields</a>, but not with complex selection controls.
```

Comment:
SKY select fields should not be used in this context. The recommendation would be just checkboxes or HTML select fields.

Reply:
Updated